### PR TITLE
Always scale ensemble evaluation points.

### PR DESCRIPTION
**Pull request by jgosmann**
Addresses #564.

### CODE REVIEW

File: nengo/tests/test_ensemble.py [hunk 1]
```diff
@@ -233,6 +233,24 @@ def heuristic(neurons, dims):
     assert points.shape == (heuristic(neurons, dims), dims)
 
 
+@pytest.mark.parametrize('eval_points, radius', [
+    (UniformHypersphere(), 0.5),
+    (UniformHypersphere(), 1.),
+    (UniformHypersphere(), 1.5),
+    (UniformHypersphere().sample(500, 1), 0.5),
+    (UniformHypersphere().sample(500, 1), 1.),
+    (UniformHypersphere().sample(500, 1), 1.5)])
```
Comment (jgosmann) [inline nengo/tests/test_ensemble.py]: Yeah, that would be better.

**Comment by hunse** [comment]
Okay, got this rebased and cleaned up that test (it's also deterministic, now). @tbekolay, want to give it a quick check and merge?

Also, this kind of pre-empts #596 now, since Connection uses the same eval point generation function as ensemble so it has to have the same scaling.

**Comment by tbekolay** [comment]
You beat me to it! Giving it a look-see now.

Re #596, the only additional stuff in that PR is the test for connection eval_points. Should we still include it? I could just squash it into this PR so it's one commit. Thoughts?

**Comment by hunse** [comment]
Yeah, maybe just put it into this PR. I think the test is less necessary now, since we're using the same function for both Ensemble and Connection eval points, so it's hard to have different scaling as we did before, but we might as well keep the test.

**Pull request merged**